packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -368,19 +380,26 @@ export class WorkspaceCommandContribution implements CommandContribution {
         return new WorkspaceRootUriAwareCommandHandler(this.workspaceService, this.selectionService, handler);
     }
 
+    protected async validateFileRename(name: string, parent: FileStat, initialValue: string, allowOverwrite: boolean = false): Promise<string> {
+        if (allowOverwrite) {
+            return '';
+        }
+        return this.validateFileName(name, parent, false);
+    }
+
     /**
      * Returns an error message if the file name is invalid. Otherwise, an empty string.
      *
      * @param name the simple file name of the file to validate.
      * @param parent the parent directory's file stat.
      * @param recursive allow file or folder creation using recursive path
```

Comment:
This has to be updated since it is renamed in the method

Reply:
Hi, unfortunately, I didn't understand what you meant, can you explain it to me ?

Reply:
@OmarSdt-EC You renamed the `recursive` argument to `allowNested`, but didn't change the documentation to reflect that. At least I think that's what he meant.